tools/topology/topology2/include/components/dummywov.conf

```diff
@@ -0,0 +1,95 @@
+#
+# Common widget dummyWoV
```

Comment:
please call this wov_test ot detect_test. We want to use inlusive terms and "dummy" isnt one of those

Reply:
OK